docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
-Manage unpackaged configuration
+Manage Unpackaged Configuration
 ===============================
 
-Deploy metadata after installing a package
-------------------------------------------
+Not everything that's part of an application can be part of a package.
 
-Namespace injection
+CumulusCI implements the Product Delivery Model by offering support for complex applications -- applications that may include multiple managed packages as well as unpackaged metadata, and setup automation that configures org settings or makes precise changes to existing configuration.
+
+The tools used to implement that support are *unpackaged metadata* and *Metadata ETL*. 
+
+Unpackaged metadata refers to metadata that is not delivered as part of a package, and can include both support metadata delivered to users as well as metadata that operationally configures orgs used by the product. 
+
+Metadata ETL is a suite of tasks that support surgically altering existing metadata in an org. It's a powerful technique that changes the unpackaged configuration in an org without risking damage by overwriting existing customizations with stored metadata. Metadata ETL is relevant for delivering applications to customers safely, and is often a superior alternative to unpackaged metadata.
```

Comment:
Would we call it a feature rather than a technique? And the end of this sentence is a bit confusing: Do the existing customizations have stored metadata? Or does Metadata ETL use stored metadata to not damage existing customizations? (I’m  95% certain it’s the former, but I would like confirmation.)